pyhf/interpolators/__init__.py

```diff
@@ -29,6 +30,7 @@ def get(interpcode, do_tensorized_calc=True):
         1: code1 if do_tensorized_calc else _slow_code1,
         2: code2 if do_tensorized_calc else _slow_code2,
         4: code4 if do_tensorized_calc else _slow_code4,
+        '4p': code4p if do_tensorized_calc else _slow_code4p,
```

Comment:
@kratsg @lukasheinrich Is there any strong motivation to not just making things _very_ explicit and call this `4piecewise` or `4piece` to try to make things quite clear given the naming confusion?

Reply:
Notational short-hand in code is very different from the long-form we'll do in the documentation.

Reply:
I think we've talked about this before, but I don't see the need to abbreviate much in names like this. But this isn't a hill I find worth dying on. :)

Reply:
if you can find two names that are non-abbreviated with the same number of characters ;)